react/components/ModalContent.tsx

```diff
@@ -1,20 +1,39 @@
 import React from 'react'
+import styles from '../styles.css'
+import classnames from 'classnames'
```

Comment:
Update your `eslint-config-vtex` via `yarn upgrade`. Import order is not being automatically fixed.